modules/form-field/css/lib/form-field.scss

```diff
@@ -67,13 +73,34 @@
   @include wdc-form-group();
 }
 
-@mixin wdc-form-inline-labels() {
+@mixin wdc-form-label-position-left() {
   .wdc-form-field-wrapper {
     display: flex;
 
-    // Case where there's no label on the left
+    &::after {
+      content: '';
+      flex-grow: 999;
+    }
+
+    .wdc-form-group-fields {
+      margin-top: -2px;
+      margin-bottom: 26px;
+    }
+
     & > .wdc-form-field {
       margin-left: $wdc-form-inline-label-width + $wdc-spacing-s;
+      // Case where there's no label on the left
+      flex-grow: 0;
+
+      .wdc-form-checkbox:first-child,
+      .wdc-form-radio:first-child {
+        margin-top: $wdc-spacing-xxxs;
+
+        &.wdc-form-alert,
+        &.wdc-form-error {
+          margin-top: 0;
```

Comment:
The padding here is incorrect in both CSS & React. I know you just matched what was in React, but can you double check it with the specs and update both?